cpp/ql/src/semmle/code/cpp/internal/QualifiedName.qll

```diff
@@ -59,7 +59,7 @@ class Namespace extends @namespace {
   }
 }
 
-abstract class Declaration extends @declaration {
+class Declaration extends @declaration {
```

Comment:
As this is internal it's not strictly necessary to remove this one, but if it doesn't break anything I don't see why we shouldn't remove it 👍

Reply:
Yeah, I agree, it shouldn't change anything, except that the class doesn't show up in the grep anymore - so why not.